src/vcf/assembly_checker.cpp

```diff
@@ -63,9 +73,14 @@ namespace ebi
 
               for (auto &output : outputs ) {
                   if (match_result) {
-                      output->write_match(record_core);
+                      std::string match_result = "Line " + std::to_string(line_num) + ": Reference allele \'"
+                          + reference_sequence + "\' matches with the fasta_sequence";
```

Comment:
We will only write mismatches to the report, otherwise we would add a lot of noise and hugely increase its size.

Reply:
dropped this line.